content/news/2017-01-stats-post.md

```diff
@@ -0,0 +1,73 @@
+The current release has been out for almost three months now. Time to take a look at some statistics!
+
+At the time when this post was drafted, there have been a recorded 53953 downloads from the official OpenRA homepage. Not unsurprisingly, most downloaded packages are for the Windows platform (almost 75%!), with Linux being a distant second at 13% and OS X coming last with the remaining 12%. It is notable that the shares of both Linux and OS X are much larger than their respective share of the desktop market.
+
+When launching OpenRA for the first time, players get greeted by a question whether to send system information back to the OpenRA project &mdash; the very data which is used to compile these statistics! Of these more than 50k downloads, almost 22,000 individuals (thanks to everyone of you!) opted in to this data gathering, which comes out to about 40%.
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-downloads-vs-users.png)
+
+Linux users are a bit more likely to enable the sysinfo gathering than the average, OS X users less so.
+</div>
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-enable-sysinfo-checkbox.png)
+
+If you like this post and happen to have OpenRA configured so that is doesn't send system information,<br /> you can easily re-enable it in the advanced settings dialog.
+</div>
+
+When the game is allowed to send system information, the time of the last updated is recorded in our database. This allows us to see how "loyal" players are to OpenRA:
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-last-launches.png)
+
+It is an unfathomable idea, but it sure looks like there are users who don't play OpenRA every day!
+</div>
+
+We can also get a general idea of the composition of our player base by looking at the language settings that get sent back to us. As expected, the vast majority of players speak English (~57%), followed by German-speaking people (16%), which combined make up almost three-fourth of all our users. Surprisingly, the relatively small Netherlands come in at third place with 4.5% of the total. Also noteworthy is a quiet Chinese minority of ~2% at seventh place.
```

Comment:
Nitpicky I know, but maybe better to say "use English as their primary language" instead of "speak English".  100% of our players speak at least enough english to have installed and run the game :smile: